src/com/projectjava/winrune/WinRune.java

```diff
@@ -351,6 +351,55 @@ public void eventDispatched(AWTEvent event) {
         gameApplet.init();
         gameApplet.start();
         frame.setSize(Constants.CLIENT_WIDTH.get(), Constants.CLIENT_HEIGHT.get() + 2); // trigger a resize to readjust positions
+
+
+
+        // Grab all stdout the gameApplet can produce to find when it's safe to set the port
+        PrintStream stdoutCapture = new PrintStream(new OutputStream() {
+          // Do not debug this section with System.out.println, since this captures System.out.println. You will get a Stack overflow!
+          StringBuilder sb = new StringBuilder();
+          String usingDefaultLoad = "Using default load" + System.lineSeparator();
+          int logCount = 0;
+          int logsBeforeSettingPort = 11;
+

```

Comment:
suggest declaring ```PrintStream sysOut = System.out;``` to use in the console print below

Reply:
Works great, thanks :+1: